sos/report/__init__.py

```diff
@@ -879,13 +879,13 @@ def list_plugins(self):
                 if opt.name in ('timeout', 'postproc', 'cmd-timeout'):
                     continue
```

Comment:
Can we also add 1 more condition to show if for example plugin.timeout is changed?

in meaning sos report -l -o dnf -k dnf.timeout=400 do not show that dnf.timeout is different.

My proposal is 

```
                if opt.name in ('timeout', 'postproc', 'cmd-timeout'):
                    if opt.value == opt.default:
                        continue
```

Reply:
Ack, good idea.